src/main/java/io/kokuwa/maven/helm/util/DependencyOverwriter.java

```diff
@@ -72,7 +72,7 @@ public void execute(Path directory) throws MojoExecutionException {
 		}
 		List<Dependency> dependencies;
 		try {
-			dependencies = MAPPER.readValue(chartFile.toFile(), Dependencies.class).getDependencies();
+			dependencies = MAPPER.readValue(chartFile.toFile(), HelmChart.class).getDependencies();
```

Comment:
this is just trivial refactoring since we've renamed the class that uses the yaml mapper